src/Feldspar/Onnx/onnxToFeld.hs

```diff
@@ -169,8 +169,23 @@ int2elemT :: Integral a => a -> TD.DataType
 int2elemT i = toEnum $ fromIntegral i :: TD.DataType
 
 showElemT :: TD.DataType -> String
-showElemT TD.FLOAT = "Float"
-showElemT t = error $ "Type " ++ show t ++ " not implemented"
+showElemT TD.FLOAT      = "Float"
+showElemT TD.UINT8      = "Word8"
+showElemT TD.INT8       = "Int8"
+showElemT TD.UINT16     = "Word16"
+showElemT TD.INT16      = "Int16"
+showElemT TD.INT32      = "Int32"
+showElemT TD.INT64      = "Int64"
+showElemT TD.STRING     = "String"
+showElemT TD.BOOL       = "Bool"
+showElemT TD.FLOAT16    = error $ "onnxToFeld.showElemT: FLOAT16 not implemented"
+showElemT TD.DOUBLE     = "Double"
+showElemT TD.UINT32     = "Word32"
+showElemT TD.UINT64     = "Word64"
+showElemT TD.COMPLEX64  = error $ "onnxToFeld.showElemT: COMPLEX64 not implemented"
```

Comment:
Isn't this `"Complex Float"` and the next line `"Complex Double"`?

Reply:
Probably. I'll put it in.